content/pt-br/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -0,0 +1,356 @@
+---
+title: Configurando GMSA Para Pods e Contêineres Windows
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
+
+Esta página mostra como configurar [Contas de serviço gerenciadas em grupo](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) (GMSA) 
+para Pods e contêineres que vão executar em nós Windows. Contas de serviço gerenciadas em grupo 
+são um tipo específico de conta do Active Directory que provê gerenciamento automático 
+de senhas, gerenciamento simplificado de *service principal name* (SPN), e a habilidade 
+de delegar o gerenciamento a outros administradores através de múltiplos servidores.
+
+No Kubernetes, especificações de credenciais GMSA são configuradas dentro do escopo 
+do cluster Kubernetes como recursos personalizados. Os Pods Windows, assim como contêineres 
+individuais dentro de um Pod, podem ser configurados para usar as funções GMSA 
+baseadas em domínio (exemplo: autenticação Kerberos) quando interagirem com outros 
+serviços Windows.
+
+## {{% heading "prerequisites" %}}
+
+Você precisa ter um cluster Kubernetes, e a ferramenta de linha de comando `kubectl` 
+precisa estar configurada para comunicar-se com seu cluster.
+O cluster deve possuir nós trabalhadores Windows. 
+Esta seção cobre o conjunto inicial de passos requeridos para cada cluster:
+
+### Instale o `GMSACredentialSpec CRD`
+
+Uma [CustomResourceDefinition](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)(CRD) uma especificação de recursos de credencial GMSA precisa ser configurada no cluster, para definir o tipo de recurso do cliente `GMSACredentialSpec`. Faça o download da GMSA CRD [YAML](https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/gmsa-crd.yml) 
```

Comment:
A few suggestions here:
* add "para" after "Uma CustomResourceDefinition(CRD)".
* "Faça o download do YAML da GMSA CRD" might be more idiomatic.